WindowsServiceManager/WindowsServiceManagerV5/DeployWindowsService.ps1

```diff
@@ -325,8 +310,23 @@ $invokeCommandSplat = @{
         $InstallService
 }
 
+if ($Machines)
+{
+    $newPSSessionSPlat = @{
+        ComputerName = $Machines 
+        SessionOption = $sessionOption 
+        UseSSL = $UseSSL
+    }
+    if ($credential)
+    {
+        $newPSSessionSPlat.Credential = $credential
+    }
+    $sessions = New-PSSession @newPSSessionSPlat    
+    $invokeCommandSplat.Session = $sessions
+
     # Import utility script into session
     Invoke-Command @invokeCommandSplat -FilePath $UTILITY_PATH
```

Comment:
Since this is inside the machines if block it will not run if it is a deployment group. Won't that break the agent deployment?
```suggestion
    Invoke-Command @invokeCommandSplat -FilePath $UTILITY_PATH
```

Reply:
I'm just testing with Deployment Groups. I don't have an Environment which us Agents for remote Deployment. So i just can say works on my Environment. Since last Improvements everything works fine for me.